src/index.njk

```diff
@@ -33,7 +33,7 @@ layout: default
 <div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32 sm:px-0">
     <div class="container m-auto text-center max-w-4xl">
         <div class="m-auto mt-36 mb-12 sm:mt-64 sm:mb-16 w-full text-center">
-            <h2>Run Node-RED in Production</h2>
+            <h2>What FlowForge brings to Node-RED</h2>
```

Comment:
@joepavitt recommended the verb `add`. I'm not a fan of `add`, as I'd rather amplify and extend Node-RED. Add seems too much like Node-RED is forked IMO